client/__tests__/e2e/e2e.test.ts

```diff
@@ -505,7 +509,7 @@ test("pan zoom mode resets lasso selection", async ({ page }) => {
     page,
     true
   );
-  expect(page.getByTestId("lasso-element")).toBeVisible();
+  expect(await page.getByTestId("lasso-element")).toBeVisible();
```

Comment:
oh wait, `page.getByTestId("lasso-element")` is a locator and doesn't return a promise!

Let's do:

```ts
await expect(page.getByTestId("lasso-element")).toBeVisible();
```